app/models/user.rb

```diff
@@ -57,6 +59,14 @@ def email_activate!
     save!(validate: false)
   end
 
+  def enforce_password_validation
+    @enforce_password_validation = true
```

Comment:
What is about this method?

Reply:
@fooooxmr When I tried to update user, I had password validation error because password was blank and I found this solution. Maybe it'll be better to add validation for password only for create, but I'm not sure if it's correct.

Reply:
I removed it